client/network/src/protocol/generic_proto/handler/group.rs

```diff
@@ -647,7 +647,8 @@ impl ProtocolsHandler for NotifsHandler {
 						..
 					}) => {
 						self.pending_legacy_handshake = Some(received_handshake);
-						break;
+						cx.waker().wake();
```

Comment:
```suggestion
						cx.waker().wake_by_ref();
```

Getting the following error otherwise:

```
error[E0507]: cannot move out of a shared reference
   --> client/network/src/protocol/generic_proto/handler/group.rs:650:7
    |
650 |                         cx.waker().wake();
    |                         ^^^^^^^^^^ move occurs because value has type `std::task::Waker`, which does not implement the `Copy` trait

   Compiling pallet-session v2.0.0-rc6 (/home/mxinden/code/github.com/paritytech/substrate/frame/session)
error: aborting due to previous error
```